### PR TITLE
Updated netty versions to 4.1.50.Final

### PR DESCRIPTION
**Pull request by kushagraThapar**
Vulnerabilities Reported:
 
* https://vuln.whitesourcesoftware.com/vulnerability/CVE-2020-11612/
 
* https://vuln.whitesourcesoftware.com/vulnerability/CVE-2019-20445/
 
* https://vuln.whitesourcesoftware.com/vulnerability/CVE-2019-20444/

### CODE REVIEW

File: sdk/cosmos/tests.yml [hunk 1]
```diff
@@ -55,71 +54,6 @@ jobs:
 
       TestResultsFiles: '**/junitreports/TEST-*.xml'
 
-
-  # This template parllelizes tests that could be run on Single_Region_Session
-  # by running them against a different cosmosdb instance
-  - template: ../../eng/pipelines/templates/jobs/archetype-sdk-tests.yml
-    parameters:
-      TestName: Single_Region_Session_2
```
Comment (moderakh) [inline sdk/cosmos/tests.yml:63]: you seem to be removing our integration tests. are the tests not working for v3?

Comment (kushagraThapar) [reply sdk/cosmos/tests.yml:63]: They are: 
* just clubbed some of different CIs to single CI with both Https and Tcp protocol. 
* removed some of the extra ones which we don't need every single time for every change. 
* We might have to do somewhat similar with master tests as well.

File: sdk/cosmos/tests.yml [hunk 2]
```diff
@@ -22,7 +22,6 @@ jobs:
           DisplayName: ReadMyWrites Integration Tests
           OSVmImage: 'ubuntu-18.04'
           ProfileFlag: '-Pe2e'
-          AdditionalArgs: '-DargLine="-Dcosmos.directModeProtocol=Https"'
```
Comment (moderakh) [inline sdk/cosmos/tests.yml:25]: why these ones are removed? is that intentional?

File: sdk/cosmos/tests.yml [hunk 3]
```diff
@@ -190,7 +189,6 @@ jobs:
           DisplayName: Single Region ReadMyWrites
           OSVmImage: 'ubuntu-18.04'
           ProfileFlag: '-Pe2e'
-          AdditionalArgs: '-DargLine="-Dcosmos.directModeProtocol=Https"'
```
Comment (moderakh) [inline sdk/cosmos/tests.yml:193]: ditto

File: sdk/cosmos/tests.yml (hunk 2 above)
Comment (kushagraThapar) [reply sdk/cosmos/tests.yml:25]: Yes, currently on V3-  we are not running e2e tests on Tcp mode, which is important as compared to running e2e tests on https mode.

File: sdk/cosmos/tests.yml (hunk 3 above)
Comment (kushagraThapar) [reply sdk/cosmos/tests.yml:193]: Yes, currently on V3- we are not running e2e tests on Tcp mode, which is important as compared to running e2e tests on https mode.

**Pull request merged**